gqlalchemy/loaders.py

```diff
@@ -227,18 +327,52 @@ def load_data(
                 yield batch
 
 
-@dataclass(frozen=True)
-class NameMappings:
+def get_data_loader(
+    file_extension: str,
+    filesystem_type: str,
+    **kwargs
+    ) -> DataLoader:
     """
-    Class that contains new label name and all of the column name mappings for a single table
+    Returns DataLoader object, which uses instance of FileSystemHandler, in order to load data
+    of specific type from specific File System.
 
-    :param label: New label (Optional)
-    :type label: str
-    :param column_names_mapping: Dictionary containing key-value pairs in form ("column name", "property name") (Optional)
-    :type column_names_mapping: Dict[str, str]
+    :param file_extension: File type to read
+    :type file_extension: str
+    :param filesystem_type: Type of filesystem we want to use
+    :type filesystem_type: str
+    :param **kwargs: For filesystem use
+
+    :returns: DataLoader 
     """
-    label: Optional[str] = None
-    column_names_mapping: Optional[Dict[str, str]] = None
+    if (file_extension == "parquet" or
+        file_extension == "csv" or
+        file_extension == "orc" or
+        file_extension == "ipc" or
+        file_extension == "feather" or
+        file_extension == "arrow"):
+        return PyarrowDataLoader(
+            file_extension=file_extension,
+            file_system_handler=get_filesystem(filesystem_type, **kwargs)
+        )
+    else:
+        raise ValueError(file_extension)
```

Comment:
`raise ValueError(f"PyArrow does not support loading {file_extension} type of files") 
`
or something like that